package.json

```diff
@@ -46,7 +46,7 @@
     "jquery-i18next": "1.2.0",
     "js-md5": "0.6.1",
     "jwt-decode": "2.2.0",
-    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#4e67cdbcbf8418d58220c9b2b1495469f3b3161e",
+    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#71afb6e0fd1d056dbaf698260b351af31c8075b7",
```

Comment:
I guess this can be dropped? I haven't seen a PR with the necessary commit though, can you make one?

Reply:
Yes, as I've mentioned in the description it depends on jitsi/lib-jitsi-meet@71afb6e which adds getConnection method. I'll adjust that once I create appropriate PR in LJM and it gets merged eventually.